src/react/components/common/filePicker/filePicker.tsx

```diff
@@ -2,11 +2,20 @@ import React, { SyntheticEvent } from "react";
 import shortid from "shortid";
 import HtmlFileReader from "../../../../common/htmlFileReader";
 
+/**
+ * Properties for File Picker
+ * @member - Function to call on change of file selection
+ * @member - Function to call on file picking error
+ */
```

Comment:
names of functions - onChange and onError - missing here.

Reply:
Good catch :)